src/style.css

```diff
@@ -0,0 +1,97 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+  --blue-accent: #00acdc;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+
+}
+
+.product-card {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
you won't need this positioning if you will use display flex for the parent element. Here you just need align-self: center after all.

Reply:
I could have done it, but in the comment higher I explained why not. But your idea seems good, thanks for it